src/goto-symex/symex_builtin_functions.cpp

```diff
@@ -532,37 +522,33 @@ void goto_symext::symex_macro(
 {
   const irep_idt &identifier=code.op0().get(ID_identifier);
 
-  if(identifier==CPROVER_MACRO_PREFIX "waitfor")
-  {
-    #if 0
-    exprt new_fc("waitfor", fc.type());
+  PRECONDITION(identifier == CPROVER_MACRO_PREFIX "waitfor");
```

Comment:
Check with @mgudemann @kroening whether we can get rid of the code here (all `#if 0` below).